kong/plugins/rate-limiting/migrations/000_base_rate_limiting.lua

```diff
@@ -108,7 +108,7 @@ return {
         period      text,
         identifier  text,
         value       counter,
-        PRIMARY KEY ((route_id, service_id, api_id, identifier, period_date, period))
```

Comment:
Why removing those? They serve the purpose of specifying a compound partition key, and are essential. I recall similar attempt to remove them already in the past - unfortunately we cannot add comments in this migration...

Reply:
Oh, it really looked like a tupo as it wasn't found anywhere else. I even tried to find this on C* docs, but couldn't. I should have tried harder. Thanks for educating me on this. I will fix it. Siirry.